### PR TITLE
Docs: Components which require a Blazorise license

### LINKED ISSUE #5797: Docs: Components which require a Blazorise license
**Is your feature request related to a problem? Please describe.**
I was wondering why my `LineChart` was not showing all the data I provided.  I tried debugging, checking what goes over the wire, reading the documentation, but nothing seemed to work. After looking at the Blazorise source code, it seems that the amount of data you provide to charts (and some other components) get limited based on your license. Our license is in the process of being renewed, so our data got cut off.

```
   internal const int DEFAULT_UNLICENSED_LIMIT_DATAGRID_MAX_ROWS = 1000;
   internal const int DEFAULT_UNLICENSED_LIMIT_AUTOCOMPLETE_MAX_ROWS = 1000;
   internal const int DEFAULT_UNLICENSED_LIMIT_CHARTS_MAX_ROWS = 10;
   internal const int DEFAULT_UNLICENSED_LIMIT_LISTVIEW_MAX_ROWS = 1000;
   internal const int DEFAULT_UNLICENSED_LIMIT_TREEVIEW_MAX_ROWS = 100;
```

**Describe the solution you'd like**
Write clearly at the top of the component's documentation what its limitations are if you do not have a license. This would have saved us some time figuring out the issue.

### Additional context
Image

### PR DESCRIPTION
**Pull request by tesar-tech**
## Description

Closes #5797

- Alert in every limited component
  image
- pricing table uses the constants from `BlazoriseLicenseProvider` (these should be in sync). Thus the table has changed its values. New table:
  image

### Question

- the `TransferList` has not the constant in `BlazoriseLicenseProvider` and its limitation doesn't appear to be enforced. So:
  - (a) Remove the TransferList from pricing table.
  - (b) Create the constant and enforce the limitation. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist:

- [x] The PR is submitted to the correct branch (`master` for dev, `rel-1.x` for maintenance).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant tests.

### CODE REVIEW

File: Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs [hunk 1]
```diff
@@ -43,88 +43,84 @@ public sealed class BlazoriseLicenseProvider
...
             backgroundWorker.RunWorkerAsync();
         }
     }
-
     #endregion
 
     #region Methods
-
-    private async void BackgroundWorker_DoWork( object sender, DoWorkEventArgs e )
+    private async void BackgroundWorker_DoWork(object sender, DoWorkEventArgs e)
```
Comment (stsrki) [inline Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs:71]: Please fix the space formatting. The format settings should be picked by the `.editorconfig` automatically.

File: Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿
+<Alert Color="Color.Warning" Visible>
+    📊 Community license has a limitation of @LimitText.
```
Comment (stsrki) [inline Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor:3]: The message should mention the limitation, but it should also be tailored in a way that promotes our paid tiers.

Maybe something like "The Community License has limitations of X. To unlock full features with our paid tiers—see pricing for details."

File: Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor
Comment (stsrki) [inline Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor:1]: Rename to LicenseLimitationAlert.razor?

Also move it under the `./Blazorise.Docs/Components/` folder?

File: Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs (hunk 1 above)
Comment (tesar-tech) [reply Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs]: fixed, wasn't turned on..

Comment (tesar-tech) [reply Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs]: it figured out "new" stuff, which I am not sure if goes from editor config or my ide setup, please re-check that.

File: Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor (hunk 2 above)
Comment (tesar-tech) [reply Documentation/Blazorise.Docs/Pages/Docs/Extensions/LicenseLimitation.razor]: Now with:
image
Shorter and talk less about "money".

**Review by stsrki — APPROVED** [review]

**Pull request merged**